floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -141,11 +142,14 @@ class QueryMethodWriter implements Writer {
     @nullable final String arguments,
     @nonNull final String mapper,
   ) {
-    final entityName = _queryMethod.queryable.name;
-
+    final queryableName = _queryMethod.queryable.name;
+    final isView = (_queryMethod.queryable is View).toString();
```

Comment:
💡The explicit `toString()` shouldn't be required as the parameter is being used with string interpolation which automatically invokes `toString()` on the boolean field.
```suggestion
    final isView = _queryMethod.queryable is View;
```